transforms/deprecate-create/deprecate-create.js

```diff
@@ -0,0 +1,143 @@
+    j.jsxAttribute(j.jsxIdentifier("iconType"), j.literal("plus")),
+    j.jsxAttribute(j.jsxIdentifier("iconPosition"), j.literal("after")),
+    j.jsxAttribute(j.jsxIdentifier("fullWidth")),
+  ];
+
+  /*
+    Change import from 'import Create from "carbon-react/lib/components/create";'
+    to 'import Button from "carbon-react/lib/components/button";'
+  */
+  const ImportReplacement = () => {
+    const createImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/create",
+      },
+    });
+
+    const buttonImport = root.find(j.ImportDeclaration, {
+      source: {
+        type: "Literal",
+        value: "carbon-react/lib/components/button",
+      },
+    });
+
+    // If there's already a Button import, just remove the Create one
+    if (buttonImport.size()) {
+      createImport.remove();
+      return true;
+    }
+
+    if (createImport.size()) {
+      createImport.replaceWith(
+        j.importDeclaration(
+          [j.importDefaultSpecifier(j.identifier("Button"))],
+          j.literal("carbon-react/lib/components/button")
+        )
+      );
+      return true;
+    }
+  };
+
+  const OpeningTagReplacement = (tag) => {
+    // Change the tag from Create to Button
+    const name = tag.find(j.JSXIdentifier, { name: "Create" });
+    if (name.size()) {
+      // If there's already attributes, add the new ones after
+      const currentAttributes = tag.find(j.JSXAttribute);
+
+      if (currentAttributes.size()) {
+        const lastAttribute = currentAttributes.last();
+        lastAttribute.insertAfter(newProps);
+        name.replaceWith(j.jsxIdentifier("Button"));
+      } else {
+        const selfClosing = tag.nodes()[0].selfClosing;
+        tag.replaceWith(
+          j.jsxOpeningElement(j.jsxIdentifier("Button"), newProps, selfClosing)
+        );
+      }
+      return true;
+    }
+  };
+
+  const ClosingTagReplacement = (tag) => {
+    // Change the tag from Create to Button
+    const name = tag.find(j.JSXIdentifier, { name: "Create" });
+
+    if (name.size()) {
+      name.replaceWith(j.jsxIdentifier("Button"));
+      return true;
+    }
+  };
+
+  const JSXElementReplacement = (create) => {
+    const importReplaced = ImportReplacement();
+    let tagsReplaced = false;
+
+    // Opening tag
+    const createOpeningTags = create.find(j.JSXOpeningElement, {
+      name: {
+        type: "JSXIdentifier",
+        name: "Create",
+      },
+    });
+
+    // Closing tag. If it doesn't exist, the opening tag is self closing
+    const createClosingTags = create.find(j.JSXClosingElement, {
+      name: {
+        type: "JSXIdentifier",
+        name: "Create",
+      },
+    });
+
+    const openingTag = createOpeningTags.last();
+    if (createClosingTags.size()) {
+      const closingTag = createClosingTags.last();
+      tagsReplaced =
+        OpeningTagReplacement(openingTag) && ClosingTagReplacement(closingTag);
+    } else {
```

Comment:
We always run `OpeningTagReplacement` we could refactor this to be 

```js
    let tagsReplaced = OpeningTagReplacement(createOpeningTags.last());

    if (createClosingTags.size()) {
      const closingTag = createClosingTags.last();
      tagsReplaced = tagsReplaced && ClosingTagReplacement(closingTag);
    }

    return tagsReplaced;
```

Reply:
Done